tokio/tests/coop_budger.rs

```diff
@@ -0,0 +1,77 @@
+#![warn(rust_2018_idioms)]
```

Comment:
Should this file be called `core_budger.rs`? Should it not be `core_budget.rs` (notice the "t" at the end and not "r")?

Reply:
Ah, nice catch. Do you want to submit a PR that fixes it?

Reply:
Admittedly not, my apologies. Just a drive-by observation as I am clocking out of work.

Reply:
#6234

Reply:
once again, the keyboard on the new laptop has bested me